client/landing/stepper/declarative-flow/site-setup-flow.ts

```diff
@@ -191,7 +194,7 @@ const siteSetupFlow: Flow = {
 			navigate( 'processing' );
 
 			// Clean-up the store so that if onboard for new site will be launched it will be launched with no preselected values
-			resetOnboardStoreWithSkipFlags( [ 'skipPendingAction' ] );
+			resetOnboardStoreWithSkipFlags( [ 'skipPendingAction', 'skipIntent' ] );
```

Comment:
Added `skipIntent` flag to prevent the site intent from being deleted before its used in the setPendingAction  function above.

Reply:
I dont see this reset being called in many other places. So it seems like we are adding `skipIntent` to the main case here?

Reply:
Yes im adding it here to ensure Im still able to retrieve the site intent value from redux inside the `exitFlow` pendingActions code.

Without adding this and modifying redux, I noticed that site intent value was always an empty string when I used the `getIntent` selector.